src/main/java/Info.java

```diff
@@ -0,0 +1,54 @@
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.*;
+
+
+public class Info {
+    private String namesJsonFilePath;
+    private Scanner scanner = new Scanner(System.in);
+    private ObjectMapper mapper = new ObjectMapper();
+    private HashMap<Integer, Names> namesMap = new HashMap<Integer, Names>(); //cr
+
+    public Info(String namesJsonFilePath) {
+        this.namesJsonFilePath = namesJsonFilePath;
+    }
+
+    public HashMap<Integer, Names> convertTomap() throws IOException, ParseException {
```

Comment:
You could consider performing this logic within the class constructor so you don't have to explicitly call this method